Sources/Accord.Statistics/TimeSeries/ACF.cs

```diff
@@ -0,0 +1,87 @@
+// Accord Statistics Library
+// The Accord.NET Framework
+// http://accord-framework.net
+//
+// Copyright © César Souza, 2009-2017
+// cesarsouza at gmail.com
+//
+//    This library is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 2.1 of the License, or (at your option) any later version.
+//
+//    This library is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public
+//    License along with this library; if not, write to the Free Software
+//    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+//
+
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Accord.Math    
+

```

Comment:
Does the code currently compile? I am afraid static methods need to be contained inside a class. It can be a static class for now, since all methods are also static.

Reply:
This code compiles in MSFT production enviornment but I still need to test it within the FRAMEWORK enviornment. I will give feedback later.

Reply:
update: Build successful using VS2017